packages/engine/crawlers/solana/index.ts

```diff
@@ -43,7 +44,9 @@ export const solanaEngineRunner: SolanaRunner = {
 				const tokenType = getTokenType(account);
 
 				if (tokenType === TokenType.Fungible) {
-					fungiblePromises.push(getSPLTokenDocument(context, pubkey, account));
+					fungiblePromises.push(
```

Comment:
Should we rename it to `tokenPromises`? (different from collections and collectibles)

Reply:
collectibles and collections is non-fungible token, so I guess it's fine. However, I suggest `splPromises`